README.md

```diff
@@ -25,7 +25,7 @@ npx @metamask/module-lint
 ### Setup
 
 - Install the current LTS version of [Node.js](https://nodejs.org)
-  - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
+  - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm install` will install the latest version and running `nvm use` will automatically choose the right node version for you.
```

Comment:
This wording is copied from our module template. Do you think we ought to change it there too? https://github.com/MetaMask/metamask-module-template/blob/65c50327f33bffb5498e06ab7a7a7feb88b9439c/README.md#L51.

I always thought that `nvm use` would automatically install the right version for you if it is not installed already. But perhaps that is not the case. What is your experience? Or do you simply think this is more clear?

Reply:
Thanks! This looks good as well. Thanks for making this change.